pwndbg/dbg/__init__.py

```diff
@@ -13,6 +13,10 @@
 dbg: Debugger = None
 
 
+class Error(Exception):
```

Comment:
Is there any real benefit for doing that instead of using pure `Exception`?

Reply:
pwndbg often distinguishes between a debugger-API-type exception and other types of exceptions. We could forego exceptions for error handling entirely in the Debugger-agnostic API, but there's a lot of spots in pwndbg where it expects error conditions to be signaled through debugger-API-specific exceptions.